src/EnergyPlus/EconomicTariff.cc

```diff
@@ -2711,7 +2711,7 @@ void ComputeTariff(EnergyPlusData &state)
 
     Real64 annualAggregate;
 
-    if (!(state.files.outputControl.tabular || state.files.outputControl.sqlite)) {
+    if (!(state.files.outputControl.tabular || state.files.outputControl.sqlite || state.files.outputControl.json)) {
```

Comment:
Just adding to the logic that is used for SQLite output.

Reply:
Yeah, it seems like this was definitely leaving too early.  But what happens here if you have:

```
Output:JSON,
  TimeSeries,
  ...
```

In your input file.  That will set `outputControl.json` to true, right?  So then what if you have tabular turned off in the `OutputControl:Files` object?  What will happen in this function?  It _looks_ like the .json variable will be true, so this will try to write tabular outputs.  Even though we didn't want it.  

My question boils down to: Is there other protections that will make sure this works in this corner case?  Or do we need to create/check a different outputControl flag (json_tabular) to ensure we are specifically not wanting tabular outputs?